homeassistant/components/beewi_smartclim/sensor.py

```diff
@@ -0,0 +1,187 @@
+"""Platform for beewi_smartclim integration."""
+import logging
+
+from datetime import datetime, timedelta
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    CONF_NAME,
+    CONF_MONITORED_CONDITIONS,
+    CONF_MAC,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_BATTERY,
+)
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+# Default values
+DEFAULT_NAME = "BeeWi SmartClim"
+
+# Sensor config
+SENSOR_TYPES = {
+    "temperature": [DEVICE_CLASS_TEMPERATURE, "Temperature", TEMP_CELSIUS],
+    "humidity": [DEVICE_CLASS_HUMIDITY, "Humidity", "%"],
+    "battery": [DEVICE_CLASS_BATTERY, "Battery", "%"],
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_MAC): cv.string,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)): vol.All(
```

Comment:
We don't allow monitored conditions in the config anymore. Please see:
https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md